### ISSUE TITLE
Update content to the latest version from PACT-kit

### ISSUE THREAD

**Issue by jmatsushita** [PanicInitiative/panicbutton.io#7]
Look at all updates from Aug 25 in [commits](https://github.com/the-engine-room/PACT-kit/commits/gh-pages)

**Comment by naturalshine** [comment]
Just to clarify -- does this simply mean that all of the material in the above-linked commits gets pasted into the existing files in _posts/app, _post/activities, etc? Is there anything more complex with this issue (and is manual transfer the best method, or is there an automated route I'm missing?)?

**Comment by jmatsushita** [comment]
Yes that's correct! Manual transfer is necessary because I've changed the posts structure from the original PACT repo.

**Comment by naturalshine** [comment]
Got it :)

**Comment by naturalshine** [comment]
Ready for review with one note: 

There's a problem with [this commit](https://github.com/the-engine-room/PACT-kit/commit/acd17576657b0f60a113b1bfcc3ec0bfa8acd9d8) from 25 August, which modifies [this](http://panicinitiative.github.io/panicbutton.io/pact-activities/pact-activities-02-risk-mapping.html).

The update inserts  *"/_posts/PACT-activities/photo%204.JPG"*
But photo%204 DNE in this directory, or anywhere else. 

Also, the checked border box at the bottom of the post wipes out .md formatting on the final bold heading, so I've changed the location of this div to allow this section designation to appear bold. Let me know if there are any problems with these adjustments !

**Comment by jmatsushita** [comment]
Waiting on answer from @tanoca

**Comment by jmatsushita** [comment]
@jimmylogan can you ask @Tanoca or make an executive decision that we don't need that image and close the issue :)

**Comment by jimmylogan** [comment]
@jmatsushita sent a question via email earlier in the week. Will leave a couple of days for response!
